src/styles/index.css

```diff
@@ -139,6 +139,29 @@
     --p-color-bg-2: var(--p-color-bg-0);
     --p-color-bg-floating: #393942;
 
+    /* Root Tokens */
+    --p-color-divider: #51525C;
+
+    --p-color-active: #1F647A;
+    --p-color-focus: #175CD3;
+
+    --p-color-awaiting: #F79008;
+    --p-color-live: #4B5CFF;
+
+    --p-color-event: #7210EF;
+    --p-color-event-dense: #A564F9;
+
+    --p-color-sentiment-positive: #22C55E;
+    --p-color-sentiment-neutral: #70707B;
+    --p-color-sentiment-negative: #D92D21;
+
+    /* Text */
+    --p-color-text-default: #FFFFFF;
+    --p-color-text-subdued: #ADADAD;
+    --p-color-text-inverse: var(--p-color-bg-0);
```

Comment:
If we changed the color of bg-0 would we want this text to match it? If not I'd just put the hex value here.

Reply:
I set the inverse to the lite mode text color and vise versa, rather than the background color. 👍